app/views/indexes/_editor.haml

```diff
@@ -4,7 +4,7 @@
     %td.even= f.text_field :name, placeholder: "Index Name", class: 'text'
   %tr
     %th.sub Owner
-    %td.odd= @inder.user.deleted? ? '(deleted user)' : @index.user.username
+    %td.odd= @index.user.deleted? ? '(deleted user)' : @index.user.username
```

Comment:
Eep! good job finding and fixing that!

Reply:
well we'd've gotten an error email if anyone had tried to load the page and we haven't, sooooo I don't think this has had a ton of impact :joy: but yes it was useful!